src/_reducers/TradesReducer.js

```diff
@@ -9,18 +9,27 @@ import {
     CREATE_TRADE,
     SERVER_DATA_PROPOSAL,
     REMOVE_PERSONAL_DATA,
+    UPDATE_TRADE_PROPOSAL,
+    UPDATE_TRADE_PURCHASE_INFO,
+    UPDATE_TRADE_UI_STATE,
 } from '../_constants/ActionTypes';
 
 const defaultTrade = {
-    symbol: 'R_100',
-    tradeCategory: 'risefall',
-    duration: 5,
-    durationUnit: 't',
-    basis: 'stake',
-    amount: 50,
-    type: 'CALL',
-    barrierType: 'relative',
-    showAssetPicker: false,
+    params: {
+        symbol: 'R_100',
+        tradeCategory: 'risefall',
+        duration: 5,
+        durationUnit: 't',
+        basis: 'stake',
+        amount: 50,
+        type: 'CALL',
+        barrierType: 'relative',
+    },
+    uiState: {
+        showAssetPicker: false,
```

Comment:
This is now a local state for the picker and nothing to do with Reducers I think.